<launch>
  <node pkg="os32c" type="os32c" name="os32c">
    <param name="hostname" value="192.168.1.1" />
    <param name="frame_id" value="os32c" />
    <param name="start_angle" value="2.3596851486963333" />
    <param name="end_angle" value="-2.3596851486963333" />
  </node>
</launch>